shopelectro/tests/tests_commands.py

```diff
@@ -130,6 +131,20 @@ def test_update_or_create_tags(self):
         self.assertEqual(updated_groups_count + create_count, TagGroup.objects.count())
         self.assertEqual(updated_tags_count + create_count, Tag.objects.count())
 
+    def test_creation_deactivated_product(self):
```

Comment:
this test should be placed at `tests_models.py` (new one, i suppose) file.
It has nothing common with commands instead of some out-of-code historical context